storage/ndb/src/kernel/vm/Configuration.cpp

```diff
@@ -358,9 +358,11 @@ Configuration::get_total_memory(const ndb_mgm_configuration_iterator *p,
     struct ndb_hwinfo *hwinfo = Ndb_GetHWInfo(false);
     return hwinfo->hw_memory_size;
     total_memory_set = false;
+    globalData.theIsRunningInContainerFlag = hwinfo->is_running_in_container;
   }
   else
   {
+    globalData.theIsRunningInContainerFlag = false;
```

Comment:
It seems you want `globalData.theIsRunningInContainerFlag` to be true if memory management should use container limit, and this can be false even if we run in a container. If correct, then this is a deceptive variable name and should be changed.